src/posh-git.psm1

```diff
@@ -18,16 +18,17 @@ if (!$Env:HOME) { $Env:HOME = "$Env:USERPROFILE" }
 $IsAdmin = Test-Administrator
 
 # Probe $Host.UI.RawUI.WindowTitle to see if it can be set without errors
-$WindowTitleSupported = $false
+$HostSupportsSettingWindowTitle = $false
 try {
-    $global:PreviousWindowTitle = $Host.UI.RawUI.WindowTitle
-    $newTitle = "${global:PreviousWindowTitle} "
+    $global:PoshGitOrigWindowTitle = $Host.UI.RawUI.WindowTitle
```

Comment:
Does this actually need to be global? Could it be script-scoped alongside `$HostSupportsSettingWindowTitle`, with a corresponding `Reset-WindowTitle` to use in `OnRemove`?

Reply:
Yeah, it could be script scoped.